corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -127,18 +134,35 @@ def handle(self, app_name, **options):
                     show_apply_commit=not has_changes
                 )
 
+        self.no_split = options.get('no_split')
         self.skip_all = options.get('skip_all')
+        if self.skip_all and self.no_split:
```

Comment:
Curious why this is? You can review the changes in the code, especially if you're not auto-committing.

Reply:
It's because we prompt the user to update the relevant views with the `@use_bootstrap5` decorator after each file is migrated in-place.

Reply:
and `--skip-all` by default auto-commits everything

Reply:
ah, I blanked on what all `skip-all` does